indigo_app/views/workflows.py

```diff
@@ -57,7 +57,7 @@ def get(self, request, *args, **kwargs):
 
         # initial state
         if not params.get('state'):
-            params.setlist('state', ['open', 'assigned', 'pending_review', 'done', 'cancelled'])
+            params.setlist('state', ['open', 'assigned', 'pending_review', 'done', 'cancelled', 'blocked'])
```

Comment:
There's an opportunity here (and maybe elsewhere) to do something like `... list(Task.STATES) + ['pending_review']` so that there's less typing of magic strings.